app/models/article.rb

```diff
@@ -876,7 +867,7 @@ def future_or_current_published_at
   end
 
   def correct_published_at?
-    return unless changes["published_at"]
+    return false unless changes["published_at"]
```

Comment:
I see this is a style change, and shouldn't affect the behaviour but technically it's `true` (published_at is correct if it hasn't changed), so `false` is a bit confusing.

Reply:
That's interesting. This was `rubocop`'s idea, which makes me think it only recently gained this preference and we maybe ought to more thoroughly examine whether it belongs with us or not. Also: this was returning `nil` before, which is `falsy`, so I can see why `rubocop` thought it might be a good idea.

Anyway, for now, I'll have it return `true`, since that seems like the best, and maybe we can bring up the `rubocop` preference in another venue.

Reply:
I've read information from rubocop and had the feeling that it was like "return false by default (instead of nil)", because in other cases we would have true if we needed it.